internal/oidc/oidc.go

```diff
@@ -186,7 +181,7 @@ type TimeoutsConfiguration struct {
 
 // Get the defaults for the Supervisor server.
 func DefaultOIDCTimeoutsConfiguration() TimeoutsConfiguration {
-	accessTokenLifespan := 15 * time.Minute
+	accessTokenLifespan := 5 * time.Minute
```

Comment:
We might want this to be even shorter (2 minutes?). That way when the Concierge client certificate expires, it's very deterministic what happens next (it always does a refresh flow).

If they are the exact same timeout, the behavior might depend on the exact clock skews between the various components, which shouldn't be buggy but would be non-deterministic and might be harder to debug.